core/node/state_keeper/src/io/seal_logic/mod.rs

```diff
@@ -339,7 +335,7 @@ impl L2BlockSealCommand {
     /// the bootloader enters the "tip" phase in which it can still generate events (e.g.,
     /// one for sending fees to the operator).
     ///
-    /// `l2_shared_bridge_addr` is required to extract the information on newly added tokens.
+    /// `l2_legacy_shared_bridge_addr` is required to extract the information on newly added tokens.
```

Comment:
I am not sure it is required (it is not present for the new chains for instance)

Reply:
tbh this comment looks out of place completely (`l2_legacy_shared_bridge_addr` is not passed/used around), I will remove it